app/views/businesses/show.html.erb

```diff
@@ -3,11 +3,9 @@
   <%= @business.description %>
 </div>
 
-<% if current_user %>
-  <% if current_user.admin? || current_user.platform_admin? %>
+<% if current_admin? || platform_admin? %>
```

Comment:
ah ha, thank you, I was looking in the application controller but didn't realize the method already existed in User